### PR TITLE
es6-arrow-functions - Marshall Vogt

### PR DESCRIPTION
**Pull request by marshall-vogt**
Screenshot 2023-08-21 at 1 37 11 PM

### CODE REVIEW

**Comment by marshall-vogt** [comment]
Screenshot 2023-08-21 at 1 37 11 PM

File: es6-arrow-functions/main.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+console.log('\n\nExercise 2');
+function actOnEntries(obj, action) {
+  Object.entries(obj).forEach((e) => action(e));
+}
+actOnEntries(book, (entry) => console.log(`${entry[0]} = ${entry[1]}`));
+
+console.log('\n\nExercise 3');
+function foo() {
+  console.log("Why doesn't this log???");
+}
```
Comment (shawnkost) [inline es6-arrow-functions/main.js]: This should become an arrow function

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Marshall

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**